docs/tutorials/build-a-mock-robot.md

```diff
@@ -304,18 +304,6 @@ This could be anything, but let's say for the purpose of this tutorial that this
 
 Navigate to your new part's **CONFIG** page and create a new motor using the **fake** model.
 
-### How to add your sub-part as a remote
```

Comment:
@edaniels We can definitely axe this portion right?

Reply:
yes